CODING-STYLE.md

```diff
@@ -0,0 +1,10 @@
+# Coding Style
+
+Our project follows the [standard Kotlin coding conventions](https://kotlinlang.org/docs/coding-conventions.html)
+with the following changes
+
+- We allow up to 120 characters per line, please use all of the available space.
+- Only capitalize the first letter when using CamelCasing, e.g. it's `UiTest`, not `UITest`,
+  `IsoMdocType`, not `ISOMDocType`, and so on.
+- Always use curly braces for conditionals. Only exception is an if-else that fits entirely on
```

Comment:
Remove this one? It looks consistent with the coding conventions listed above (it's not an exception). https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements indicates that multi-line conditionals should always use curly braces.

Reply:
OK.